tutorials/javaFxPart4.md

```diff
@@ -81,14 +81,25 @@ Note that if you are using packages, `fx:controller="MainWindow"` needs to be up
 
 <fx:root alignment="TOP_RIGHT" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" prefWidth="400.0" type="javafx.scene.layout.HBox" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
   <children>
-    <Label fx:id="dialog" text="Label" wrapText="true" />
+    <Label fx:id="dialog" text="Label" wrapText="true" minHeight="-Infinity"/>
     <ImageView fx:id="displayPicture" fitHeight="99.0" fitWidth="99.0" pickOnBounds="true" preserveRatio="true" />
   </children>
   <padding>
     <Insets bottom="15.0" left="5.0" right="5.0" top="15.0" />
   </padding>
 </fx:root>
 ```
+<box type="info" seamless>
+
+Note that for `Label`, we set `wrapText` to **true** and `minHeight` to **-Infinity**. This configuration wraps the text within the label, removes the minimum height constraint, and allows the label to grow vertically as needed.
+
+![Set minHeight for Label](images/javafx/NoOverrunDialogBox.png)
+
+Without setting `minHeight` to **-Infinity**, text overrun may occur when the text to be displayed exceeds the size of the label, causing the text to not be fully displayed and ending with `...` instead.
+
+![No minHeight for Label](images/javafx/OverrunDialogBox.png)
+
+</box>
```

Comment:
Perhaps we can make it easy to spot the problem in the second screenshot?
e.g.,
![image](https://github.com/user-attachments/assets/47d8f899-7eab-4d3c-8b75-7368ad3e248a)

Also, try to make the example screenshots not bigger than needed, so that they don't consume valuable screen space more than they need to.

Reply:
I have updated the second screenshot and cropped them such that each screenshot only shows one complete `DialogBox` .